models/DataObject/ClassDefinition/Data.php

```diff
@@ -404,6 +404,12 @@ public function getFilterConditionExt(mixed $value, string $operator, array $par
             return $key . ' ' . $operator . ' ' . $value . ' ' . $trailer;
         }
 
+        if ($operator === 'in') {
```

Comment:
@mfedak-notebooksbillger Thank you for the PR. Looks good. Just a small refactoring. This code block can be moved to the check above the line https://github.com/pimcore/pimcore/blob/c82dfa2e72095d98ec6a9d4e90974acdf2467157/models/DataObject/ClassDefinition/Data.php#L375 so that we can avoid the extra checks  for the 'in' operator in https://github.com/pimcore/pimcore/blob/c82dfa2e72095d98ec6a9d4e90974acdf2467157/models/DataObject/ClassDefinition/Data.php#L381 and https://github.com/pimcore/pimcore/blob/c82dfa2e72095d98ec6a9d4e90974acdf2467157/models/DataObject/ClassDefinition/Data.php#L389